cpp/src/arrow/testing/json_integration_test.cc

```diff
@@ -54,6 +54,9 @@ DEFINE_string(
     "Mode of integration testing tool (ARROW_TO_JSON, JSON_TO_ARROW, VALIDATE)");
 DEFINE_bool(integration, false, "Run in integration test mode");
 DEFINE_bool(verbose, true, "Verbose output");
+DEFINE_bool(no_decimal_validate, false,
```

Comment:
It's generally nicer to avoid double negatives, so I'd prefer `DEFINE_bool(validate_decimals, true, "...")`